release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -158,6 +158,13 @@ In {product-title} {product-version}, you can use Confidential VMs when installi
 Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
 ====
 
+[id="ocp-4-13-installation-aws-local-zones"]
+==== Installing a cluster on AWS into an existing Virtual Private Cloud (VPC) improvements
+
+In {product-title} {product-version}, the installation process for clusters that use AWS VPCs is simplified. This release also introduces the _edge pool_, a pool of machines that are optimized for AWS Local Zones. 
+
+For more informsation, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster using AWS Local Zones].
```

Comment:
@maxwelldb 
s/informsation/information/

Reply:
Thanks! Translators should catch this in what we sent off on Thursday. I'll make an English change tomorrow.